DokuwikiXMLExport.php

```diff
@@ -50,71 +49,117 @@ class DokuwikiXMLExport
     const PROPERTY_DUMMY_VALUE = array('dummy');
 
     /**
-     * Generate the entire XML Export based on the Dokuwiki metadata.
+     * @var array $conf DokuWiki configuration.
+     */
+    protected $conf;
+
+    /**
+     * @var array $pages All pageIds.
+     */
+    protected $pages;
+
+    /**
+     * @var int $key Amount of loops.
+     */
+    protected $key;
```

Comment:
`$pages` makes sense as an instance variable, `$key` not so much.

Reply:
Okay. I wasn't sure if this makes sense or not. Thanks!